tests/suites/test_suite_cipher.function

```diff
@@ -950,6 +950,14 @@ void test_vec_crypt( int cipher_id, int operation, data_t *key,
     if( use_psa == 1 )
     {
         PSA_ASSERT( psa_crypto_init( ) );
+
+        if (cipher_id == MBEDTLS_CIPHER_AES_192_ECB ||
```

Comment:
I'm a bit confused now. I understood that ECB should support only 128 bit size (and return error otherwise):

> Note that for ECB, unlike other modes and unlike psa_cipher_xxx, mbedtls_cipher_crypt for ECB only accepts 128-bit inputs. So for inputs of different lengths, we expect an error (but we still want to have a test, to ensure that we aren't getting garbage output or a crash).

Reply:
That's 128-bit *inputs*. The “ECB” functions in the cipher module actually only encrypt a single block, and the size of a block is always 128 bits for AES. The number in `MBEDTLS_CIPHER_AES_192_ECB` refers to the size of the *key*, and all three possible sizes for AES (128, 192 or 256 bits) are supported.